tests/pytorch/test_numerics.py

```diff
@@ -40,11 +40,13 @@
 from transformer_engine.pytorch.attention.inference import InferenceParams
 from transformer_engine.pytorch.distributed import checkpoint as te_checkpoint
 from transformer_engine.pytorch.cpp_extensions import general_gemm, general_grouped_gemm
+from transformer_engine.pytorch.cpp_extensions.fused_attn import FusedAttnBackend
 from transformer_engine.pytorch.tensor.float8_tensor import Float8Quantizer
 from transformer_engine.pytorch.module.base import get_multi_stream_cublas_workspace, get_workspace
 from transformer_engine.pytorch.utils import get_device_compute_capability, get_cudnn_version
 from transformer_engine.common import recipe
 import transformer_engine_torch as tex
+from utils import ModelConfig, _rng_states, reset_rng_states, get_available_attention_backends
```

Comment:
No need to import `_rng_states` unless we want to do something with it. It's an implementation detail of `reset_rng_states`, so importing it should be viewed as an evil hack.
```suggestion
from utils import ModelConfig, reset_rng_states, get_available_attention_backends
```
We import it in a few other files.